packages/qwik/src/core/components/prefetch.unit.tsx

```diff
@@ -24,15 +24,17 @@ describe('PrefetchServiceWorker', () => {
       expect(output.html).to.contain('<script nonce="1234" q:key="prefetch-service-worker">');
     });
     it('should render script with a scope', async () => {
+      // if the qwik app was isolated to /en/ folder
+      // scope will only run in /en/ pathname in the url
       const output = await renderToString(
-        <PrefetchServiceWorker base="/build/en/" scope="/en/" />,
+        <PrefetchServiceWorker base="/en/build/" scope="/en/" />,
```

Comment:
this contradicts the "service worker not in build" comment

Reply:
well the problem is that base in this case is q:base which is /build/ not / so the / is /en/

Reply:
:no_mouth: I'm confused now